app/components/shared/ShowWarning.js

```diff
@@ -1,5 +1,5 @@
 const ShowWarning = ({ warn, className }) => (
-  <div className={"warning" + (className ? (" " + className) : "")}>{warn}</div>
+  <div className={"warning " + className}>{warn}</div>
```

Comment:
If  `className` is ever `undefined` or `null` the resulting `className` in the div ends up including a wrong value. Either revert the change or do a `["warning", className].join(" ")`.

Reply:
fixed